gohelper.go

```diff
@@ -35,9 +35,12 @@ type Credential struct {
 	Type     string `json:"type"`
 	Port     int    `json:"port"`
 	Hostname string `json:"hostname"`
-	Query    struct {
-		IsMaster bool `json:"is_master"`
+	// Split for Credential.Query.IsMaster included to handle inconsistent
+	//  JSON-encoding of `json:"query"`.
+	Query      struct {
+		IsMaster bool `json:"-"`
 	}
+	RawQuery   interface{}  `json:"query"`
```

Comment:
This should probably be `rawQuery` so that it's not accessible outside this package.  It's an internal implementation detail.